editor/edit-post/meta-boxes.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Function returning the current Meta Boxes HTML in the editor
+ * whether the meta box area is opened or not.
+ * This is not so clear, but I believe it's the only way to have this data synchronously
+ *
+ * @param   {String} location Meta Box location
+ * @returns {String}          HTML content
+ */
+export const getLocationHtml = ( location ) => {
+	const area = document.querySelector( `.editor-meta-boxes-area.is-${ location } .editor-meta-boxes-area__container` );
+	if ( area ) {
+		return area.innerHTML;
+	}
+
+	return document.querySelector( '.metabox-location-' + location ).innerHTML;
```

Comment:
When would we expect this line to be reached? (Can we add a comment explaining this?)

Reply:
And if we're expecting the first query selector might return an empty result, is there any reason to think this one might as well?

Reply:
The metaboxes are rendered to an initial hidden dom node (the later here) but when their panel is visible (it's not always the case for the sidebar), they are moved to their new location and when closed moved back to the original location.